### PR TITLE
Feature 19 front page bottom links and post page left bar

### PR DESCRIPTION
**Pull request by Daniilmass**

### CODE REVIEW

**Comment by Daniilmass** [comment]
This PR contains saved work state including Front page bottom section grid layout with pictures and text (links) which are taken from the WP Posts. Also, this PR contains a Post page side bar component.

File: src/components/pages/WelcomePage.tsx [hunk 1]
```diff
@@ -238,6 +243,48 @@ class WelcomePage extends React.Component<Props, State> {
...
 
+  /**
+   * Render Bottom section posts
+   * 
+   */
+  private renderBottomSectionPosts = (categoryId: number) => {
+    const { classes } = this.props;
+    const linkedEventsPost = this.state.linkedEventsPost;
+    if (!this.state.posts) {
+      console.log("NO POSTS");
```
Comment (terolamminpaa) [inline src/components/pages/WelcomePage.tsx:254]:
```suggestion
```

File: src/components/pages/WelcomePage.tsx [hunk 2]
```diff
@@ -238,6 +243,48 @@ class WelcomePage extends React.Component<Props, State> {
...
+      return null;
+    } else {
+      console.log("ALL POSTS ARE ", this.state.posts);
```
Comment (terolamminpaa) [inline src/components/pages/WelcomePage.tsx:257]:
```suggestion
```

File: src/components/pages/WelcomePage.tsx [hunk 3]
```diff
@@ -238,6 +243,48 @@ class WelcomePage extends React.Component<Props, State> {
...
+      return(
+        this.getLimitedPosts(categoryId, 6).map(post => {
+          console.log("BOTTOM SECTION POST IS: ", post.title);
```
Comment (terolamminpaa) [inline src/components/pages/WelcomePage.tsx:260]:
```suggestion
```

File: src/components/pages/WelcomePage.tsx [hunk 4]
```diff
@@ -238,6 +243,48 @@ class WelcomePage extends React.Component<Props, State> {
...
+          return (
+            <div style={{ backgroundImage: `url(${this.getAttachmentForPost(post)})` }} className={classes.bottom_section_item}>
+              <p>{ReactHtmlParser(post.title ? post.title.rendered || "" : "")}</p>
```
Comment (terolamminpaa) [inline src/components/pages/WelcomePage.tsx:263]: This should probably work without react html parser? I think post.title.rendered is just a string without any html.

**Pull request merged**